1.2/spec.md

```diff
@@ -1023,128 +1044,138 @@ When possible, any mapping of logical structure onto HTML should try to follow t
   * text should be in reading order
   * all tags should be used for the intended purpose (and only for the intended
     purpose) as defined in the [[HTML40]] spec.
-  * floats are contained in `<div>` elements with a `style` that includes a float attribute
+  * floats are contained in <{div}> elements with a `style` that includes a float attribute
   * repeating floating page elements (header/footer) should be repeated and occur
     in their natural location in reading order (e.g., between pages)
   * embedded images and SVG should be contained in files in the same directory
-    (no `/` in the URL) and embedded with `<img>` and `<embed>` tags, respectively
+    (no `/` in the URL) and embedded with <{img}> and <{embed}> tags, respectively
 
 Specifically
 
-  * `<em>` and `<strong>` should represent emphasis, and are preferred to `<b>`, `<i>`, and `<u>`
-  * `<b>`, `<i>`, and `<u>` should represent a change in the corresponding
+  * <{em}> and <{strong}> should represent emphasis, and are preferred to <{b}>, <{i}>, and <{u}>
+  * <{b}>, <{i}>, and <{u}> should represent a change in the corresponding
     attribute for the current font (but an OCR font specification must still be
     given)
-  * `<p>` should represent paragraph breaks
-  * `<br>` should represent explicit linebreaks (not linebreak that happen because of text flow)
-  * `<h1>`, ..., `<h6>` should represent the logical nesting structure (if any) of the document
-  * `<a>` should represent hyperlinks and references within the document
-  * `<blockquote>` should represent indented quotations, but not other uses of indented text.
-  * `<ul>`, `<ol>`, `<dl>` should represent lists
-  * `<table>` should represent tables, including correct use of the `<th>` tag
+  * <{p}> should represent paragraph breaks
+  * <{br}> should represent explicit linebreaks (not linebreak that happen because of text flow)
+  * <{h1}>, ..., <{h6}> should represent the logical nesting structure (if any) of the document
```

Comment:
This does not work. The correct link goes to https://html.spec.whatwg.org/multipage/semantics.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements